uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/test/resources/infinispan.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ Copyright 2018 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<infinispan>
+  <cache-container>
+    <replicated-cache name="cache">
+      <indexing index="LOCAL" auto-config="true"/>
+    </replicated-cache>
+  </cache-container>
+</infinispan>
```

Comment:
This should be ``` <indexing index="ALL" auto-config="true"/>```

FYI, a ```replicated-cache``` will replicate the data to all nodes of the cluster. This has the following advantages:
* Extremely fast queries, resolved locally without contacting other nodes
* High availability, as long as only one node is up the data is safe and the index is alive

But it also has drawbacks:
* Each node holds the full dataset. Indexes are stored on disk, data is stored in memory. If you plan to run it in modest hardware, you need to estimate how much data you have to hold and see if replicated is the right config. If not, we should move to <distributed-cache/>.

Reply:
Ok I will use for now ALL to see how it behaves, thanks!